readthedocs/core/unresolver.py

```diff
@@ -76,6 +103,8 @@ class DomainSourceType(Enum):
 
 @dataclass(slots=True)
 class UnresolvedDomain:
+    # The domain that was used to extract the information from.
+    source_domain: str
     source: DomainSourceType
     project: Project
     domain: Domain = None
```

Comment:
What's the difference between `domain` and `source_domain`? We should clarify this with a comment in the code.